cpu/samd5x/cpu.c

```diff
@@ -354,7 +353,8 @@ void cpu_init(void)
 
     xosc_init(0);
     xosc_init(1);
-    fdpll0_init(CLOCK_CORECLOCK * DPLL_DIV);
+    fdpll_init(0, CLOCK_CORECLOCK * DPLL_DIV);
+    fdpll_init(1, MHZ(200));
```

Comment:
Current consumption is back to normal when I remove this line.

Reply:
Strange, the ONDEMAND bit is set and since nobody is connected yet to the associated GCLK it shouldn't impact the overall consumption...
I'll investigate.

Reply:
Okay, I've found the reason why. ONDEMAND bit is "enable-protected". 
I'll provide a fix later.
This also mean that the FDPLL0 is not running with ONDEMAND set which is a bug.